CONTRIBUTING.md

```diff
@@ -0,0 +1,235 @@
+# Contributing to Edge Agent Swift SDK
+
+:rocket::tada: First off, thanks for taking the time to contribute! :tada::rocket:
+The following guidelines are for contributing to Edge Agent Swift SDK. These are mostly guidelines. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+#### Table Of Contents
+
+[What should I know before I get started?](#what-should-i-know-before-i-get-started)
+  * [Edge Agent](#atala-prism)
```

Comment:
According to the replacement table, `Atala PRISM` should be Identus but it could also be `Identus Cloud Agent` but it seems not to e `Edge Agent`

@goncalo-frade-iohk , I have left all my comments for the review of this PR under this commit: https://github.com/input-output-hk/atala-prism-wallet-sdk-swift/commit/f5f488af1e71a6c528fe7503062b03ae6abbb3c4

Reply:
Done